src/ingest_llm_as/processors/conversation_ingestor.py

```
@@ -34,17 +37,18 @@ async def start(self):
                 if processed_count == 0:
                     await asyncio.sleep(5)
             except Exception as e:
-                logger.error(f"Error in ingestor loop: {e}")
+                logger.error(f"Error in ingestor loop: {e}", exc_info=True)
```

Ruff (0.14.11)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)